packages/iov-crypto/src/random.ts

```diff
@@ -0,0 +1,27 @@
+declare const window: any;
+declare const self: any;
+
+export class Random {
+  /**
+   * Returns `count` cryptographically secure random bytes
+   */
+  public static getBytes(count: number): Uint8Array {
+    try {
+      const globalObject = "object" === typeof window ? window : self;
```

Comment:
I was momentarily confused by this because I think I've only ever seen "typeof x === y" never the other way around.

Reply:
Oh, yeah this is an artifact from external code. Will update since I don't like it as well.